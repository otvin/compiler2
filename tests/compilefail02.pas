program compilefail02(output);
var
	a:integer;
begin
	a:=3.14;
	writeln(a);
end.
