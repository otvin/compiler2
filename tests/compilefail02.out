Cannot assign value '3.14' of type real literal to type integer in file:tests/compilefail02.pas line:5     column:8  
